src/components/ClusterOverview/Alerts/Alerts.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../Dashboard/DashboardCard';
+import { ClusterOverviewContext } from '../ClusterOverviewContext';
+import { AlertsBody } from '../../Dashboard/Alert/AlertsBody';
+import { filterAlerts } from './utils';
+import { AlertItem } from '../../Dashboard/Alert/AlertItem';
+
+export const Alerts = ({ alertsResponse }) => {
+  if (!Array.isArray(alertsResponse)) {
+    return null;
+  }
+
+  const alerts = filterAlerts(alertsResponse);
+
+  return alerts.length > 0 ? (
+    <DashboardCard className="kubevirt-alerts__card">
```

Comment:
the top border probably shouldn't go to Alerts card

Reply:
Im passing className to `Alerts` from `ClusterOverview` now